content/cpp/concepts/math-functions/terms/sin/sin.md

```diff
@@ -0,0 +1,45 @@
+---
+Title: 'sin()'
+Description: 'Returns the sine of an angle given in radians in the range [-1,1].'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Functions'
+  - 'Arithmetic'
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+Returns the sine of an angle (argument) given in radians in the range [-1,1].
+
+## Syntax
+
+```cpp
+std::sin(n)
+```
+
+The argument `n` is an angle given in radians.
+
+## Example 
+
+The following example illustrates the `sin()` function:
+
+```cpp
+#include<iostream>  
+#include<math.h>  
+using namespace std;  
+int main()  
+{  
+  double degree = 60;  
+  //convert degree to radian
+  double radian = degree*3.14/180;  
```

Comment:
To improve this output, I would suggest utilizing the `M_PI` constant (more precise than 3.14):

```suggestion
  // convert degrees to radians
  double pi = M_PI;
  double degree = 60;  
  double radian = degree * pi/180;  
```